elastic.py

```diff
@@ -1,71 +1,83 @@
-
-    def displayMainWindow(self):
-        tab = "Main Window"
-
-        (Shuffleboard.getTab(tab)
-        .add("Note Detected!", self.isNoteDetected)
-        .withSize(4,4)
-        .withPosition(17,0)
-        .withWidget(BuiltInWidgets.kBooleanBox)
-        .getEntry())
-
-        (Shuffleboard.getTab(tab)
-        .add("Note on Left", self.isNoteLeft)
-        .withSize(4,4)
-        .withPosition(13,0)
-        .withWidget(BuiltInWidgets.kBooleanBox)
-        .getEntry())
-
-        (Shuffleboard.getTab(tab)
-        .add("Note on Right", self.isNoteRight)
-        .withSize(4,4)
-        .withPosition(21,0)
-        .withWidget(BuiltInWidgets.kBooleanBox)
-        .getEntry())
-
-        (Shuffleboard.getTab(tab)
-        .add("Note Loaded!",self.isNoteLoaded)
-        .withSize(4,4)
-        .withPosition(17,4)
-        .withWidget(BuiltInWidgets.kBooleanBox)
-        .getEntry())
+        self.startingPositions = activeStartingPositions
+        self.isElasticSubmitted = False
+        self.controllerDriverElastic = False
+        self.controllerOperatorElastic = False
+        self.beamBreakBoolean = False
+    
+    def teamDisplay(self, defaultColor, defaultLabel):
+        self.teamChooser = wpilib.SendableChooser()
+        color = (defaultColor + " Alliance")
+        label = defaultLabel
+        self.teamChooser.setDefaultOption(color, label)
+        if defaultColor == 'Red':
+            self.teamChooser.addOption("Blue Alliance", "BLUE")
+        else:
+            self.teamChooser.addOption("Red Alliance", "RED")
+        SmartDashboard.putData(self.teamChooser)
+    
+    def positionDisplay(self, defaultPosition):
+        self.positionChooser = wpilib.SendableChooser() 
+        position = ("Positon " + defaultPosition)
+        defaultLabel = defaultPosition
+        self.positionChooser.setDefaultOption(position, defaultLabel)
+        for i in self.startingPositions:
+            position = ("Positon " + i)
+            label = i
+            if not(defaultLabel == label):
+                self.positionChooser.addOption(position, label)
+        SmartDashboard.putData(self.positionChooser)
 
     def autonDisplay(self):
         self.chooser = wpilib.SendableChooser()
         self.chooser.setDefaultOption(self.autonPlans[0], self.autonPlans[0])
-
         for plan in self.autonPlans:
             self.chooser.addOption(plan, plan)
-
         SmartDashboard.putData(self.chooser)
+    
+    def updateBeamDisplay(self, noteDetected):
+        if noteDetected:
+            self.beamBreakBoolean = True
+        elif not(noteDetected):
+            self.beamBreakBoolean = False
+        else:
+            None
+        self.putBoolean("NOTE LOADED", self.beamBreakBoolean)
+
+    def updateControllerConnectedDisplay(self, driverConnected, operatorConnected):
+        if driverConnected:
+            self.controllerDriverElastic = True
+            self.putBoolean("Driver Connected", self.controllerDriverElastic)
```

Comment:
Similar to the above, could the arguments just get passed directly so that the `if/else` pairs aren't needed?